agent.go

```diff
@@ -67,6 +67,10 @@ const (
 	cannotGetTimeMsg   = "Failed to get time for event %s:%v"
 	pciBusRescanFile   = "/sys/bus/pci/rescan"
 	pciBusMode         = 0220
+	// If a process finish because fatal error with  signal "n"
```

Comment:
s/finish/terminates/
s/because fatal error with signal/because of signal/

Reply:
Changed :)